config/v1/types_infrastructure.go

```diff
@@ -354,8 +354,21 @@ type AWSPlatformSpec struct {
 	// There must be only one ServiceEndpoint for a service.
 	// +optional
 	ServiceEndpoints []AWSServiceEndpoint `json:"serviceEndpoints,omitempty"`
+
+	// LBType allows user to set a load balancer type.
+	// Allowed values are NLB or Classic
+	// +kubebuilder:validation:Enum:=NLB;Classic
```

Comment:
For backwards compatibility you need to allow an empty value too
```suggestion
	// +kubebuilder:validation:Enum:=NLB;Classic;""
```